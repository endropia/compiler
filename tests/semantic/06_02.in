var
	a: integer;

begin
	writeln(a.b);
end.