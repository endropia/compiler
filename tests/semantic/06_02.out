(5, 10) It is not record